README.md

```diff
@@ -14,15 +14,18 @@ The command above will create a local directory with a dump of the MTC state.
 
 You will get a dump of:
 - All namespaces where a MTC toolset is installed, including pod logs
-- All velero.io and migration.openshift.io resources located in those namespaces
+- All velero.io and migration.openshift.io resources located in MTC namespaces
+- StorageClasses and PersistentVolume resources
+- Route and Service resources from the `default` and `openshift-image-registry` namespaces
+- All resources from namespaces migrated in the most recent migration attempt, with the exception of Secrets
 - Prometheus metrics
 
 **Essential-only gather**
 
 Differences from full gather:
  - Logs are only gathered from specified time window
  - Skips collection of prometheus metrics, pprof. Removes duplicate logs from payload.
-```
+```sh
```

Comment:
is "sh" intentional here? It won't show up on README I think

Reply:
Yeah, so this bit just signals to the [markdown renderer](https://docs.github.com/en/github/writing-on-github/creating-and-highlighting-code-blocks#syntax-highlighting) that this block of code should be rendered with syntax highlighting to shell scripts. 

I believe [this is the full list](https://github.com/github/linguist/blob/master/lib/linguist/languages.yml) of supported syntax highlighting.